### PR TITLE
Add glossary entry for Git

### PR DESCRIPTION
**Pull request by adellape**
Been seeing varied usage coming in to OpenShift docs, so thought might as well get some guidance down somewhere (didn't find in IBM SG).

Other usage examples in the wild for consideration, FWIW:

* [Git docs](https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository)
* [GitHub docs](https://docs.github.com/en/github/getting-started-with-github/set-up-git)
* [Wiki entry](https://en.wikipedia.org/wiki/Git)

### CODE REVIEW

**Review by bergerhoffer — DISMISSED** [review]
LGTM, thanks for adding this!

**Comment by bergerhoffer** [comment]
@redhat-documentation/ccs-style-council Please review

**Comment by leswilliams44** [comment]
LGTM

**Review by leswilliams44 — APPROVED** [review]

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc [hunk 1]
```diff
@@ -204,7 +204,7 @@ The API object for a deployment can be either a Kubernetes-native `Deployment` o
...
+*Description*: An image is a pre-built, binary file that contains all of the necessary components to run a single container; a container is the working instantiation of an image. Additionally, an image defines certain information on how to interact with containers created from the image, such as what ports are exposed by the container. OpenShift uses the same image format as Docker; existing Docker images can easily be used to build containers through OpenShift. Additionally, OpenShift provides a number of ways to build images, either from a Dockerfile or directly from source hosted in a Git repository.
```
Comment (ndeevy) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift.adoc:207]:
```suggestion
*Description*: An image is a pre-built, binary file that contains all of the necessary components to run a single container; a container is the working instantiation of an image. Additionally, an image defines certain information about how to interact with containers created from the image, such as what ports are exposed by the container. OpenShift uses the same image format as Docker; existing Docker images can easily be used to build containers through OpenShift. Additionally, OpenShift provides a number of ways to build images, either from a Dockerfile or directly from source hosted in a Git repository.
```

Suggestion to use 'information about' not 'on' (guidance from ISG )

**Review by ndeevy — DISMISSED** [review]

**Comment by ndeevy** [comment]
LGTM!  just added one tiny nitpick suggestion.

**Comment by adellape** [comment]
@ndeevy @bergerhoffer Updated per latest review.

**Review by ndeevy — APPROVED** [review]

**Review by bergerhoffer — APPROVED** [review]

**Pull request merged**